Library/PledgeViewContext.swift

```diff
@@ -3,98 +3,106 @@ import Foundation
 public enum PledgeViewContext {
   case fixPaymentMethod
   case pledge
+  case latePledge
   case update
   case changePaymentMethod
   case updateReward
 
   var confirmationLabelHidden: Bool {
     switch self {
     case .fixPaymentMethod, .changePaymentMethod, .updateReward: return true
-    case .pledge, .update: return false
+    case .pledge, .latePledge, .update: return false
```

Comment:
What happens if you want to the update the reward of a late pledge? Is it `.latePledge` or .`updateReward?`

Reply:
You can't update rewards (or change payment method) for late pledges, afaik. Since you're charged immediately, your pledge is also locked in as soon as you make it.

Reply:
Love it.